app/src/androidTest/java/com/github/h3lp3rs/h3lp/StorageTest.kt

```diff
@@ -23,14 +25,15 @@ class StorageTest {
     private val TEST_KEY = "KEY"
     private val TEST_SEED = Random(0)
     private val BYTES_PER_CHAR = 2
-    private val DELTA = 1e-10
 
     @Before
     fun setup() {
         globalContext = ApplicationProvider.getApplicationContext()
-        Databases.PREFERENCES.db = MockDatabase()
+        userUid = USER_TEST_ID
+        PREFERENCES.db = MockDatabase()
         resetStorage()
-        stor = storageOf(MEDICAL_INFO)

```

Comment:
Weird variable name "stor* not really clear what it represent maybe can be change to stroage

Reply:
Yes will change!